crates/devnet/README.md

```diff
@@ -106,29 +108,35 @@ independent nonce.
 Then, deploy the contract.
 
 ```shell
-mpc-devnet mpc my-test deploy-contract 
+mpc-devnet mpc $MPC_NETWORK_NAME deploy-contract 
```

Comment:
Thanks! Not having a variable for this has bothered me, but not enough to open a standalone PR to fix it :sweat_smile: